tests/integration_tests/core_tests.py

```diff
@@ -1128,6 +1128,9 @@ def test_get_column_names_from_metric(self):
             "my_col"
         ]
 
+    @pytest.mark.skip(
```

Comment:
@dpgaspar I think you may have written those 2 tests before that I'm disabling here. I believe these tests were looking for a failure of some kind, but were (are at least currently were) picking up on the language pack entry for "Error message". Turns out the "bootstrap payload" is including the language pack for the current locale, and if it happens to be `en` it included a lot of empty strings with empty mappings. I'm now making sure that the `en` language pack has an empty payload as opposed to a map with empty targets.

Now i'm thinking these 2 unit tests are wrong, and maybe unneeded, so I'm skipping them here and sending this message in a bottle to figure out if we want to follow up with anything on those.